rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -52,6 +53,8 @@ def __init__(
             database=database,
             msg_aggregator=database.msg_aggregator,
         )
+        # Passing self is not good, but it is the cleanest approach found so far
+        proxies_inquirer = EthereumProxiesInquirer(ethereum_inquirer=self)
```

Comment:
you only need the `database`, `ds_proxy` contract and `multicall` in the `EvmProxiesInquirer`. Maybe you can pass just those.

Also I believe you can leave it like this. irc we do something similar in other places